iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/NullLiteralImpl.java

```diff
@@ -21,11 +21,12 @@
 
 import java.util.List;
 import org.sonar.iac.arm.tree.api.NullLiteral;
+import org.sonar.iac.arm.tree.api.bicep.HasToken;
 import org.sonar.iac.arm.tree.api.bicep.SyntaxToken;
 import org.sonar.iac.arm.tree.impl.AbstractArmTreeImpl;
 import org.sonar.iac.common.api.tree.Tree;
 
-public class NullLiteralImpl extends AbstractArmTreeImpl implements NullLiteral {
+public class NullLiteralImpl extends AbstractArmTreeImpl implements NullLiteral, HasToken {
```

Comment:
Same here. `NullLiteral` should extend `HasToken`

Reply:
Same here aswell. We only have `YamlTreeMetadata metadata` inside the json variant. Should we return null?

Reply:
Result: Leave at is.